docs/source/reference_aggregation.rst

```diff
@@ -1,30 +1,54 @@
-Other Methods of Aggregations
+Aggregation special cases
 =============================
 
 As we have already seen in this `page <https://fsspec.github.io/kerchunk/test_example.html#multi-file-jsons>`_,
-that the main purpose of ``kerchunk`` it to generate references, to view whole archive of files like
-GRIB2, NetCDF etc. allowing us for *in-situ* access to the data. In this part of the documentation,
-we will see some other efficient ways of combining references.
+that the main purpose of ``kerchunk`` it to generate references, to view whole archive
+of files like GRIB2, NetCDF etc. allowing us for direct access to the data. In
+this part of the documentation, we will see some other efficient ways of
+combining references.
 
 GRIB Aggregations
 -----------------
 
-This new method for reference aggregation, developed by **Camus Energy**, is based on GRIB2 files. Utilizing
-this method can significantly reduce the time required to combine references, cutting it down to
-a fraction of the previous duration. In reality, this approach builds upon consolidating references
-with ``kerchunk.combine.MultiZarrtoZarr``, making it faster. The functions and operations used in this
-will be a part of the ``kerchunk``'s API. You can follow this `video <https://discourse.pangeo.io/t/pangeo-showcase-optimizations-for-kerchunk-aggregation-and-zarr-i-o-at-scale-for-machine-learning/4074>`_
-for the intial discussion.
+This reference aggregation method of GRIB files, developed by **Camus Energy**, functions if
+accompanying ``.idx`` files are present.
+
+**But this procedure has some certain restrictions:**
+
+  - GRIB files must paired with their ``.idx`` files
+  - The ``.idx`` file must be of *text* type.
+  - Only specialised for time-series data, where GRIB files
+    have *identical* structure.
+  - Aggregation only works for a specific **forecast horizon** files.
```

Comment:
The reference index can be combined across many horizons but each horizon must be indexed separately.
Looking forward to seeing what you make of the reinflate api... there you can see all of the FMRC slices are supported agains a collection of indexed data from many horizons, runtimes and valid times.

Reply:
> the reinflate api

ooh, what is this?

Reply:
The method to turn the k_index and the metadata back into a ref_spec you can use in zarr/xarray
https://github.com/asascience-open/nextgen-dmac/blob/main/grib_index_aggregation/dynamic_zarr_store.py#L198
I think @Anu-Ra-g is already working on adding it into kerchunk?

Reply:
Oh, in that case I suspect it works already, right @Anu-Ra-g : but you can only work on one set of horizons OR one set of timepoints, not both at once? Something like that.

Reply:
I think you can return an array with multiple dimensions.
I didn't have a strong use for this so struggled to do something general and practical.
For instance if you request by Horizon, you can provide multiple horizon axis and you dimensions should include 'horizon' and 'valid_time". Similarly you can request multiple runtimes and then your dimensions should include 'runtime' and 'step'.
Honestly not sure if this is helpful or over complicated.

Reply:
@martindurant I tried it out with one set of horizons with the original code. Actually, I'm still figuring out the reinflating  part of the code,  aggregation types and the new indexes. 

I noticed that for reinflating can also work with a `grib_tree` model made from a **single grib** file. 
@emfdavid can you confirm this in this notebook that I made?